npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -37,6 +37,18 @@ type IPSetManagerCfg struct {
 	NetworkName string
 }
 
+var (
+	ApplyOnNeedCfg = &IPSetManagerCfg{
```

Comment:
Do we need to standardize this ? We know that IPSetMode is a enum, we do not have to define enum of the Cfg obj.

Reply:
I made variables for IPSetManagerCfg and PolicyManagerCfg unexported and only kept them for UTs